site.yml.sample

```diff
@@ -72,30 +72,35 @@
         - ansible_distribution == 'Fedora'
         - ansible_distribution_major_version|int >= 23
 
-  roles:
-    - ceph-defaults
-    - ceph-validate
-
+    - include_vars: roles/ceph-defaults/defaults/main.yml
```

Comment:
I believe this will override any variables we've set in other places as vars included in a task take precedence over vars set in the inventory or a group_vars file. This might be your problem with ``ceph_origin``.  You can see from the CI log (https://2.jenkins.ceph.com/job/ceph-ansible-prs-luminous-update_cluster/52/consoleFull) that ``ceph_origin`` is being reset to ``dummy``.

```
TASK [include_vars] ************************************************************
task path: /home/jenkins-build/build/workspace/ceph-ansible-prs-luminous-update_cluster/site.yml.sample:75
Wednesday 05 September 2018  09:33:05 +0000 (0:00:00.409)       0:00:45.308 *** 
ok: [ceph-mon0] => {
    "ansible_facts": {
        "bootstrap_dirs_group": "64045", 
        "bootstrap_dirs_owner": "64045", 
        "centos_package_dependencies": [
            "python-pycurl", 
            "epel-release", 
            "python-setuptools", 
            "libselinux-python"
        ], 
        "ceph_conf_key_directory": "/etc/ceph", 
        "ceph_conf_local": false, 
        "ceph_conf_overrides": {}, 
        "ceph_custom_repo": "https://server.domain.com/ceph-custom-repo", 
        "ceph_dev_branch": "master", 
        "ceph_dev_sha1": "latest", 
        "ceph_docker_enable_centos_extra_repo": false, 
        "ceph_docker_image": "ceph/daemon", 
        "ceph_docker_image_tag": "latest", 
        "ceph_docker_on_openstack": false, 
        "ceph_docker_registry": "docker.io", 
        "ceph_iscsi_config_dev": true, 
        "ceph_iscsi_firewall_zone": "public", 
        "ceph_keyring_permissions": "0600", 
        "ceph_mds_firewall_zone": "public", 
        "ceph_mgr_firewall_zone": "public", 
        "ceph_mirror": "http://download.ceph.com", 
        "ceph_mon_firewall_zone": "public", 
        "ceph_nfs_firewall_zone": "public", 
        "ceph_origin": "dummy", 
        "ceph_osd_firewall_zone": "public", 
        "ceph_rbdmirror_firewall_zone": "public",
 ...
```

Reply:
Agreed. I added it for debugging and later forgot to get rid of it.